src/frontend/Komponenter/Behandling/Inngangsvilkår/Aleneomsorg/Bosted.tsx

```diff
@@ -12,25 +12,23 @@ const Bosted: FC<Props> = ({ harSammeAdresseSøknad, harSammeAdresseRegister, er
     return (
         <>
             {harSammeAdresseRegister !== undefined && harSammeAdresseRegister !== null ? (
-                <>
-                    <Registergrunnlag />
-                    <BodyShortSmall>Bosted</BodyShortSmall>
-                    <BodyShortSmall>
-                        {harSammeAdresseRegister
+                <Informasjonsrad
+                    ikon={VilkårInfoIkon.REGISTER}
+                    label="Bosted"
+                    verdi={
+                        harSammeAdresseRegister
                             ? 'Registrert på søkers adresse'
-                            : 'Ikke registrert på søkers adresse'}
-                    </BodyShortSmall>
-                </>
+                            : 'Ikke registrert på søkers adresse'
+                    }
+                />
             ) : (
                 harSammeAdresseSøknad !== undefined &&
                 harSammeAdresseSøknad !== null && (
-                    <>
-                        <Søknadsgrunnlag />
-                        <BodyShortSmall>Bosted</BodyShortSmall>
-                        <BodyShortSmall>
-                            {utledVisningAvBostedVerdier(erBarnetFødt, harSammeAdresseSøknad)}
-                        </BodyShortSmall>
-                    </>
+                    <Informasjonsrad
+                        ikon={VilkårInfoIkon.SØKNAD}
+                        label="Bosted"
+                        verdi={utledVisningAvBostedVerdier(erBarnetFødt, harSammeAdresseSøknad)}
```

Comment:
Sånn jeg tolker det bruker vi `verdiSomString` når verdien ikke selv har noe styling++. Burde den da settes til true her hvor det bare en en string som sendes inn?

Reply:
Ah, så først nå at den har default value. Lurt 💯

Reply:
`verdi` skal i de fleste tilfeller bare være en label for raden, men for å gjøre `Informasjonsrad` mulig å bruke alle steder er typen til verdi `ReactNode`. `verdiSomString` settes derfor til default `true` slik at `verdi` returneres i en label. I de tilfellene hvor man vil ha annerledes styling (f.eks. kopierbart fødselsnummer) så sender man inn dette direkte i `verdi` og setter `verdiSomString=false`